src/ucs/config/parser.c

```diff
@@ -1437,8 +1437,12 @@ ucs_config_parser_set_value_internal(void *opts, ucs_config_field_t *fields,
                     return status;
                 }
             }
-        } else if (((table_prefix == NULL) || !strncmp(name, table_prefix, prefix_len)) &&
-                   !strcmp(name + prefix_len, field->name))
+        } else if (/* check exact match */
```

Comment:
I propose to add a few simple unit tests in `test_config.cc` to cover this behaviour

Reply:
good point, pls share your ideas about test cases